packages/dashboard/src/utils/usePagePreviewSize.js

```diff
@@ -97,26 +97,28 @@ const sizeFromWidth = (
 const getContainerWidth = (windowWidth) => {
   // Because the dashboard has a min width (MIN_DASHBOARD_WIDTH) check to see if that min should be used or the actual space of the dashboard
   const isWindowSmallerThanMinDashboardWidth =
-    window.innerWidth < MIN_DASHBOARD_WIDTH;
+    globalThis.innerWidth < MIN_DASHBOARD_WIDTH;
   return isWindowSmallerThanMinDashboardWidth
     ? MIN_DASHBOARD_WIDTH
     : windowWidth;
 };
 
 export default function usePagePreviewSize(options = {}) {
   const { thumbnailMode = false, isGrid } = options;
+  const container =
+    typeof document !== 'undefined' ? document.getElementById(WPBODY_ID) : null;
   // When the dashboard is pulled out of wordpress this id will need to be updated.
   // For now, we need to grab wordpress instead because of how the app's rendered
```

Comment:
Sounds like something we should refactor now and move to `wp-dashboard` somehow...

Reply:
#11276